src/JWT/TokenExpiredException.cs

```diff
@@ -1,12 +1,38 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace JWT
 {
     public class TokenExpiredException : Exception
     {
-        public TokenExpiredException(string message)
+       private const string PayloadDataKey = "PayloadData";
+       private const string ExpirationKey = "Expiration";
+
+       public TokenExpiredException(string message)
             : base(message)
         {
         }
+
+        public Dictionary<string, object> PayloadData
+        {
+            get
+            {
+               if (Data.Contains(PayloadDataKey))
```

Comment:
Sure, let's keep it safe. Can you please move it a helper? And remove a space inside the cast operator, to follow the common style.